### PR TITLE
feat: add esnext.string to ts libs

### PR DESCRIPTION
**Pull request by jake-bassett**
## Description
Added esnext.string to tsconfig libs.

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "downlevelIteration": true,
-    "lib": ["es2015", "es2016", "es2017", "esnext.array", "esnext.asynciterable", "dom"],
+    "lib": ["es2015", "es2016", "es2017", "esnext.string","esnext.array", "esnext.asynciterable", "dom"],
```
Comment (aaron-steinfeld) [inline tsconfig.base.json:19]: did you verify that we polyfill these?

Comment (jake-bassett) [reply tsconfig.base.json:19]: Not _everything_, but I tested `replaceAll`. What else should I look for?

Comment (aaron-steinfeld) [reply tsconfig.base.json:19]: we basically need to make sure that it works in any browser we support (not just local) - so either it's been implemented in all that we support, or safer - that core-js polyfills replaceall (I think that's all that comes in here, right?)

Comment (jake-bassett) [reply tsconfig.base.json:19]: Ah, that I did check.

We are good on all modern browsers. 

https://caniuse.com/?search=replaceAll

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**